Solutions/Claroty/Analytic Content/ClarotyNewAsset.yaml

```diff
@@ -0,0 +1,29 @@
+id: 6c29b611-ce69-4016-bf99-eca639fee1f5
+name: Claroty - New Asset Alert
+description: |
+  'Triggers when new asset discovered.'

```

Comment:
Can you please double check if the logic detects when critical deviation from baseline occurs or if it detects new asset discovered?

Reply:
fixed